### PR TITLE
29-created new header-components

### LINKED ISSUE #57: 29 - Create header component
[HEADER](https://www.figma.com/file/MpfNES0KuBUkkd0Qk4EyOc/HYF-Connect-Wireframe?node-id=463%3A330)

- [x] Create a div container class:
- [x] Insert the HYF-Connect logo with a link to the home page on the left side of the header
- [x] Create a centered navbar with 4 links:
- A link to `projects` page
- A link to `member` page
- A link to `about us` page
- A link to `contact us` page
- [x] A button linked to `login` page on the right side of the header

### PR DESCRIPTION
**Pull request by firity**
Removed header.html

### CODE REVIEW

File: client/components/header-component.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-  </a>
-  <a class="link" href="/pages/projects/projects.html">projects</a>
-  <a class="link" href="/pages/members/members.html">members</a>
-  <a class="link" href="/pages/about-us/about-us.html">about us</a>
-  <a class="link" href="/pages/contact-us/contact-us.html">contact us</a>
-  <button>Log In</button>
-</div>`
+    template: `<div class="header-container">
+    <div class="brand">
+            <a href="https:" class="logo"><img src="/public/assets/logo_pc_full.png" alt="logo" width="90" height="70"/></a>
```
Comment (dirk-jacobs) [inline client/components/header-component.js]: Clicking on the logo should bring us back to the homepage..
Replace the link of the logo (currently https: ??? with /pages/homepage/homepage.html or just /

File: client/components/header-component.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+            </div>
+            <div class="nav-bar">
+                <a class="nav-link" href="/pages/projects/projects.html">projects</a>
+                <a class="nav-link" href="/pages/members/members.html">members</a>
+                <a class="nav-link" href="/pages/about-us/about-us.html">about us</a>
+                <a class="nav-link" href="/pages/contact-us/contact-us.html">contact us</a>
+            </div>
+            <a class="sign-in-btn" href="/pages/sign-in.html">Sign-in</a>
```
Comment (dirk-jacobs) [inline client/components/header-component.js]: The link to the signup page is incorrect
/pages/sign-in/sign-in.html

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
We are making progress: 'Close but no cigar'
Please update the links and as a bonus: do you think it is possible to make the updates you did in the about-us.html page (google fonts + link to styles.css)  also in the members.html, projects.html and contact-us.html ?

**Review by dirk-jacobs — DISMISSED** [review]
Looks ok to me

**Review by rayanejsilva — COMMENTED** [review]
@firity I adjusted link paths in the nav-bar.

**Review by rayanejsilva — APPROVED** [review]

**Review by Moamin-AbuEwaida — APPROVED** [review]
LGTM
we'll have a conflict with the sign in naming as i changed it to login in my branch to skip the confusion 
Thanks @firity

**Pull request merged**